### ISSUE TITLE
inspect causes "RangeError: Maxim call stack size exceeded"

### ISSUE THREAD

**Issue by slicejunk** [graphql/graphql-js#1783]
[version 4.1.1]
 inspect causes RangeError in this line from "values.js"
```
       error.message =
                `Variable "$${varName}" got invalid value ${inspect(value)}; ` +
                error.message;
```
when receiving a ```value```  with coercion error that contains an object such as a file to upload, maybe due to its circular structure

**Comment by Cito** [comment]
@slicejunk can you give us some more information on which kind of file object you were using? What do you use for uploading the files? I tried some variants but none of these created circular structures. Would be good 1) to know that this really caused your problem and 2) to have an object that we can use for regression tests.

**Comment by IvanGoncharov** [comment]
@Cito You can reproduce it with:
```js
const a = {};
a.a = a;
inspect(a);
```
Node's `inspect` produces `{ a: [Circular] }`.

@slicejunk Thanks for reporting this issue 👍 
We definetly include fix in the next release (ETA sometime this week).

**Comment by IvanGoncharov** [comment]
Fixed in #1771
